draft-ietf-wpack-bundled-responses.md

```diff
@@ -85,8 +85,9 @@ Bundle parsers support two primary operations:
 ## Naming a representation {#semantics-naming}
 
 Representations within a bundle are named by their `Content-Location` (Section
-8.7 of {{!I-D.ietf-httpbis-semantics}}), which holds a URL. This is also known
-as the representation's URL.
+8.7 of {{!I-D.ietf-httpbis-semantics}}). This is also known as the
+representation's URL. It is either an absolute-URI or a partial-URI. In the
```

Comment:
I'm not certain whether to cite `absolute-URI or a partial-URI ({{Section 2.7 of RFC7230}})` or `valid url string without a fragment ({{URL}})` here, but do include the link to the specification that defines the term.

Reply:
Done